lib/dotcom/trip_plan/input_form.ex

```diff
@@ -25,7 +24,7 @@ defmodule Dotcom.TripPlan.InputForm do
   typed_embedded_schema do
     embeds_one(:from, __MODULE__.Location)
     embeds_one(:to, __MODULE__.Location)
-    field(:datetime_type, Ecto.Enum, values: @time_types)
+    field(:datetime_type, :string)
```

Comment:
📓 I changed this for now because we need to have a way to navigate between these atoms and the string we're given back by the form. We can figure this out later.

Reply:
the atom/string juggling was definitely my least favorite part about trying to handle mode changes too...